src/applications/post-911-gib-status/containers/IntroPageSummary.jsx

```diff
@@ -110,55 +106,42 @@ export default function IntroPageSummary() {
         itemScope
         itemType="http://schema.org/Answer"
       >
-        {!environment.isProduction() && (
```

Comment:
I'm pretty sure that we don't want this here anymore. Need to verify w/ @peggygannon and design, since it looks like it's only ever been feature flagged at this point, anyway